src/ontology/edits/oeo-physical.omn

```diff
@@ -14502,6 +14502,36 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1744",
         OEO_00030024
     
     
+Class: OEO_00360040
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A critical infrastructure role is a role of artificial objects, such as plants and systems, that are assessed to be crucial for the preservation of societal functions, health, safety and economical or social welfare."@en,
```

Comment:
The definition implies an axiom `'critical infrastructure role' 'has bearer' some 'artificial object'` which is missing